best-practices/README.md

```diff
@@ -220,6 +220,11 @@ Browsers
 
 * Don't support versions of Internet Explorer before IE9.
 
+DNS
+---
+
+* Avoid using non-standard DNS entries (i.e. ALIAS, ANAME).
```

Comment:
Nitpick appreciated and you are correct.

On Friday, July 24, 2015, Mike Burns <notifications@github.com> wrote:

> In best-practices/README.md
> <https://github.com/thoughtbot/guides/pull/347#discussion_r35418161>:
>
> > @@ -220,6 +220,11 @@ Browsers
> >
> >  * Don't support versions of Internet Explorer before IE9.
> >
> > +DNS
> > +---
> > +
> > +* Avoid using non-standard DNS entries (i.e. ALIAS, ANAME).
>
> Super grammatical nitpick but this feels more like an e.g. than an i.e. to
> me ("for example" instead of "that is").
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/guides/pull/347/files#r35418161>.
>